pagerduty/service.go

```diff
@@ -85,6 +85,31 @@ type Service struct {
 	Type                   string                     `json:"type,omitempty"`
 }
 
+// ServiceEventRule represents a service event rule
+type ServiceEventRule struct {
+	ID         string                      `json:"id,omitempty"`
+	Self       string                      `json:"self,omitempty"`
+	Disabled   bool                        `json:"disabled,omitempty"`
+	Conditions *RuleConditions             `json:"conditions,omitempty"`
```

Comment:
Good reuse of the `RuleConditions` type from Rulesets. There is one minor difference that we can probably validate in the provider code, and that is the `path` field needs to be a valid [PD-CEF ](https://developer.pagerduty.com/docs/events-api-v2/overview/#pagerduty-common-event-format-pd-cef )field value.